### DISCUSSION TITLE
Is there any way to edit simultaneously with a friend?

### DISCUSSION THREAD

**Discussion by Hagashi** [haven1433/HexManiacAdvance#158] [Q&A]
Is it possible to use this application with other people, making changes in a multiplayer way? And if it's not possible to do something like that?

**Comment by haven1433** [comment]
HMA isn't set up for online file sharing like Word or Excel, but there is a feature to let two people make coordinated edits.

The most important thing when making binary changes that you want to merge in later is to make sure you're each writing to different freespace. In that case, you can use patches to merge changes later, and use diffs to merge the .toml files. To do this, you just have to change your freespace address in the bottom left corner.

For example:
* Hacker 1 uses `A00000` freespace
* Hacker 2 uses `C00000` freespace
* Hacker 3 isn't repointing anything, just changing pokemon stats / item stats, but sets their freespace to `780000` just in case they need to repoint text or a moveset.
